README.md

```diff
@@ -1 +1,148 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+
+
+**[Hello Microverse Project]** This is a simple project that displays "hello microverse".Its a true reflection of skills I have acquired from microverse so far.
+## 🛠 Built With <a name="built-with"></a>
+- HTML.
+- CSS.
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+- vscode.
+- git.
+- set up an index.html document.
+
+### Setup
+Clone this repository to your desired folder:
+
+Navigate to the  directory where you  want to clone this project in your computer.
+Clone this repository to your desired folder:git clone [https://github.com/ReX0069/Hello_Microverse-.git](https://github.com/DevKaranJ/Hello-Microverse)
+
+
+### Install
+Install this project with this command:
+`$ npm install`
+
+
+### Usage
+
+To run the project, execute the following command by opening the vscode with the command `code .`
+
+### Run tests
+
+To run tests, run the following command:
+
+Run it in your local server
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Karan Joshi**
+
+- GitHub: [@githubhandle](https://github.com/DevKaranJ)
+- Twitter: [@twitterhandle](https://twitter.com/RX0069)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/dev-karan-j/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- Setting Up A Professional Github Repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] While you make a great effort 😃 I see that when I click on the link of issues pages in the contributing section the link redirects me to the page not found, so please try to fix that by putting the correct link to let the users open the issues pages and communicate with you and give you feedback about your project and with that you can enhance the projects from the visitor's feedback💯🚀